### DISCUSSION TITLE
Getting started

### DISCUSSION THREAD

**Discussion by priyanka-go-plus** [SafeExamBrowser/seb-win-refactoring#756] [Q&A] [answered]
Hello 
we are online assessment platform and trying to integrate safe exam browser for our online assessments. Where should we start from. I went through the documents and i understand that we need to first download and install it and when we are starting the assessment we should check if user is using SEB else he should be prompted to install / open exam in SEB. Am I right?

**Comment by dbuechel** [comment]
Exactly, all necessary information can be found on our website: https://safeexambrowser.org/developer/seb-integration.html.

**Reply by priyanka-go-plus** [reply]
I have implemented seb in our platform. Problem I am facing is I am getting a url to config with https scheme from api. Should I change the scheme of url to open seb browser when I open the url in window.open or I need to do something else. As right now when I open the file with window.open it downloads the config

**Comment by danschlet** [comment]
Just read that page https://safeexambrowser.org/developer/seb-integration.html properly. There is a section:

There are several ways how to open those SEB Config Files to either start an exam with the contained settings or to configure SEB clients:

By clicking (loading) a special hyperlink with the custom URL protocol scheme seb/sebs. The link should contain the address of a SEB Config File. The advantage of using a custom URL protocol is that other browsers and basically all applications which can open URLs will start SEB and open the link there, similar like opening a .seb file with SEB. But using a SEB Config Link doesn’t require to download and save the SEB Config File to the local file system. SEB itself will just load the configuration temporarily into memory, when it’s ment to start an exam. The process is also smoother with this method, applications usually just ask if it’s ok to open the link in SEB. As mentioned before, opening SEB Config Links which direct to a SEB Config File saved on a web server which requires authentication works only in specific SEB versions, currently in SEB for macOS 2.1.1 and newer and SEB for Windows 2.2. In the latter there is currently the limitation that loading a configuration file on an authenticated server works only when opening a seb/sebs link from within SEB, not from another browser/application. But otherwise  the link doesn’t even have to contain the config file name with extension .seb, it can also be an indirect link similar to mylms.com/file.php?id=34.

**Reply by priyanka-go-plus** [reply]
I have gone through it. And added seb:// to config url but gives wrong config source error

**Comment by danschlet** [comment]
- Use sebs:// when your server delivers https://
- Check out [this page](https://safeexambrowser.org/exams/index3.html) (the source code of this simple http page), there are numerous examples of sebs:// links (quit password is usually "quit" in these example .seb configurations, some of those demo exams don't work anymore). You have to find out yourself how to make it work with Javascript window.open, we're no web developers.

**Reply by priyanka-go-plus** [reply, marked as answer]
it wasnt javascript issue.SEB was throwing error since i had added seb instead sebs to scheme. thanks for all your help and time.
